app/pods/components/agenda/agenda-header/component.js

```diff
@@ -254,16 +254,20 @@ export default Component.extend(FileSaverMixin, {
     },
 
     async downloadAllDocuments() {
+      // timeout options is in milliseconds. when the download is ready, the toast should last very long so users have a time to click it
       const fileDownloadToast = {
         title: this.intl.t('file-ready'),
         type: 'download-file',
-        options: { timeOut: 10 * 1000 }
+        options: { timeOut: 60 * 10 * 1000 } 
```

Comment:
This arbitrary time is funny. It has to specified? Hmm probably has to be 🤔  `Infinity` ? No I'm just joking here...